doc/z-chap06.xml

```diff
@@ -7,28 +7,12 @@
   be created in &Semigroups;, and the options that are available at the time of
   creation.
  
-  <Section Label = "Acting semigroups"><!--TODOFIXME: move this-->
```

Comment:
Is deleting these two big chunks of text a good idea? Unless they are repeated elsewhere they should probably just be commented out for now.